carbonj.service/src/main/java/com/demandware/carbonj/service/engine/kinesis/cfgCheckPointMgr.java

```diff
@@ -0,0 +1,87 @@
+/**
+ * Copyright (c) 2018, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+package com.demandware.carbonj.service.engine.kinesis;
+
+import com.amazonaws.services.dynamodbv2.AmazonDynamoDB;
+import com.amazonaws.services.dynamodbv2.AmazonDynamoDBClientBuilder;
+import com.demandware.carbonj.service.accumulator.Accumulator;
+import com.demandware.carbonj.service.accumulator.cfgAccumulator;
+import com.demandware.carbonj.service.engine.*;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Import;
+
+import java.nio.file.Paths;
+import java.util.Date;
+import java.util.concurrent.ScheduledExecutorService;
+import java.util.concurrent.TimeUnit;
+
+@Configuration
+@Import({ cfgAccumulator.class, cfgKinesis.class })
+public class cfgCheckPointMgr {
+
+    @Value( "${metrics.store.checkPoint.dir:work/carbonj-checkpoint}" ) private String checkPointDir;
+
+    @Value( "${metrics.store.checkPoint.offset.default.mins:5}" ) private int defaultCheckPointOffset;
+
+    @Value( "${metrics.store.checkPoint.provider:filesystem}" ) private String checkPointProvider;
+
+    @Value( "${metrics.store.checkPoint.applicationName:cjajna}" ) private String checkPointApplicationName;
+
+    @Value( "${metrics.store.checkPoint.table.provisioned.throughput:2}" ) private int
+            checkPointTableProvisionedThroughput;
+
+    private static final Logger log = LoggerFactory.getLogger( cfgCheckPointMgr.class );
```

Comment:
Why not start using @Slf4j annotation which I believe comes with Spring Boot.

Reply:
that might be a topic for a different PR..